src/util/instanceBulkActions.tsx

```diff
@@ -69,7 +69,12 @@ export const instanceActionLabel = (action: LxdInstanceAction): string => {
 };
 
 export const pluralize = (item: string, count: number): string => {
-  return count === 1 ? item : `${item}s`;
+  const isSingular = count === 1;
+  if (item.includes("identity")) {
+    return isSingular ? item : item.replace("identity", "identities");
+  }
+
+  return isSingular ? item : `${item}s`;
```

Comment:
```suggestion
  if (isSingular) {
    return item;
  }
  if (item.includes("identity")) {
    return item.replace("identity", "identities");
  }
  return `${item}s`;
```
Maybe a little more straight forward?

Reply:
nice! :+1: